test/cp.js

```diff
@@ -797,7 +797,7 @@ test('should not attempt to copy fifos via symlinks in directories', t => {
 test('should copy fifos directly', t => {
   try {
     shell.exec(`mkfifo ${t.context.tmp}/fifo`);
-    shell.exec(`echo -n test1 > ${t.context.tmp}/fifo`, { async: true });
+    shell.exec(`printf test1 > ${t.context.tmp}/fifo`, { async: true });
```

Comment:
Oh yeah :-) good point :-)

Is it possible to easily shell out to shelljs's echo implementation? (I've shelled out here because the event loop is blocked by the cp, so need a way to get data into the fifo in the background; and atm this is shelling out to the /bin/sh impl (iirc?) and using it's echo/printf (although maybe there's a better way?)).